front-end/src/App.tsx

```diff
@@ -45,6 +51,38 @@ class App extends React.Component<any, any>{
       src: URL.createObjectURL(await response.blob())
     });
   }
+
+  labels = ['Duration'];
+  data = {
+  labels: this.labels,
+  datasets: [
```

Comment:
Is this dummy data for the visualization? What does the `label` stand for?

Reply:
Oh from the demo video I guess it's the label of each bar?

Reply:
Yup, it's the dummy labels for the dummy data. Since we currently don't have a way to estimate duration. We will update this once we have that ready.